multimodal/src/autogluon/multimodal/utils/metric.py

```diff
@@ -82,6 +84,9 @@ def infer_metrics(
                 validation_metric_name = HIT_RATE
             else:
                 validation_metric_name = eval_metric_name
+            if validation_metric_name == NDCG:
+                # TODO(?) Currently NDCG is not supported. We may pick a better replacement of NDCG for validation in the future.
+                validation_metric_name = HIT_RATE
```

Comment:
I think this can be removed.

Reply:
Let's do it in a follow-up PR. The logic should be correct. Basically, validation_metric_name can never be NDCG because it is not supported.